tempesta_fw/http.c

```diff
@@ -2542,7 +2549,7 @@ tfw_http_resp_process(TfwConn *conn, struct sk_buff *skb, unsigned int off)
 bad_msg:
 	bad_req = tfw_http_popreq(hmresp);
 	if (bad_req)
-		tfw_http_conn_msg_free((TfwHttpMsg *)bad_req);
+		tfw_http_send_500(bad_req, "response dropped: processing error");
```

Comment:
Now I'm fine with the change - that's good as a default behaviour. However, `bad_msg` can be reached by `TFW_BLOCK` returned from GFSM hook, typically Frang security limit. A user should have an options whether to generate HTTP response, which is relatively heavy operation, or just drop malicious traffic (filtered by Frang). This is the subject for #658 which I reassign to you. Please note the place when you'll be working on it.

Reply:
Ok